test/c3/appframework/appframework_test.go

```diff
@@ -427,15 +470,27 @@ var _ = Describe("c3appfw test", func() {
 
 	Context("Clustered deployment (C3 - clustered indexer, search head cluster)", func() {
 		It("c3, integration, appframework: can deploy a C3 SVA with apps installed locally on CM and SHC Deployer, and cluster-wide on Peers and SHs", func() {
-			// Upload V2 apps to a 2nd S3 bucket as we need 2 buckets for this test.(V1 apps to be used for local install, V2 apps for cluster install)
-			s3TestDirCluster := "c3appfw-cluster-" + testenv.RandomDNSName(4)
-			ClusterappFileList := testenv.GetAppFileList(appListV2, 2)
-			uploadedFiles, err := testenv.UploadFilesToS3(testS3Bucket, s3TestDirCluster, ClusterappFileList, downloadDirV2)
+
+			//Delete apps on S3 for new Apps to split them accorss both cluster and local
+			testenv.DeleteFilesOnS3(testS3Bucket, uploadedApps)
+			uploadedApps = nil
+
+			//Split Applist into 2 list for local and cluster install
+			appListLocal := appListV1[len(appListV1)/2:]
+			appListCluster := appListV1[:len(appListV1)/2]
+
+			// Upload appListLocal to bucket 1 on S3
+			appFileList := testenv.GetAppFileList(appListLocal, 1)
```

Comment:
Since we are already covering upgrade scenario in other test cases wouldnt it make sense to install V2 and downgrade to v1. Instead of v1 -> upgrade to v2 -> downgrade to v1 ?

Reply:
Only reason for keeping it this way was that this is the only scenario with Local + cluster wide installation of app and i wanted to keep both upgrade and downgrade scenario.

Even with this the test case is fairly quick  
• [SLOW TEST:1165.531 seconds]
c3appfw test